doc/conf.py

```diff
@@ -47,7 +47,7 @@
 
 # General information about the project.
 project = u'KATCP'
-copyright = u'2009-2012, SKA South Africa'
+copyright = u'2009-2012, National Research Foundation (South African Radio Astronomy Observatory)'
```

Comment:
In the other files we don't have an upper bound on the the copyright.  Either extend the upper limit to 2019, or remove it (the latter is easier to maintain).

Reply:
Interesting point. Copyright only seems to care about the _first_ year of publication, so it does indeed sound like a great idea to remove those pesky upper bounds 😄